assets/scss/app.scss

```diff
@@ -7,10 +7,12 @@
 // provide the scss or plain css, it only provides the less version
 // of its source files, that's why it is not included it via npm.
 @import "../vendor/bootstrap-datetimepicker.css";
-@import "../vendor/jquery-ui.css";
+@import "~jquery-ui/themes/base/sortable.css";
```

Comment:
afaik we are only using sortable from jquery-ui

Reply:
If the developper is using other features of jquery, it won't work for him.
So the BC-break requires to be documented.

Reply:
Done